static/main.html

```diff
@@ -3,9 +3,13 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="main.css" charset="utf-8">
+    <title>WebTorrent Desktop</title>
+    <link rel="stylesheet" href="main.css">
+    <script>
+	require('../build/renderer/main.js')
+    </script>
   </head>
   <body>
-    <script async src="main.js"></script>
+    <div id='body'></div>
```

Comment:
That's what I did before, but React prints a warning when you do that. (For silly reasons that don't apply to us--apparently some ppl have browser extensions that break React when used on `<body>`--but I also didn't see a better way to turn off the warning.)